src/views/Login/Login.js

```diff
@@ -131,7 +144,6 @@ export default function SignIn() {
             labelWidth={70}/>
         </InputText>
 
-
           <FormControlLabel
```

Comment:
It does not remember a logged-in user.

Reply:
I am not the one who did login integration but when I was doing my work it was working as it should.
Let me inspect to see what might be the cause.